openid-4-verifiable-presentations-1_0.md

```diff
@@ -1704,6 +1704,17 @@ Note: If the Verifier's Response URI did not return a `redirect_uri` in step (6)
 
 (10) The Verifier checks whether the `nonce` received in the Credential(s) in the VP Token in step (9) corresponds to the `nonce` value from the session. The Verifier then consumes the VP Token and invalidates the `transaction-id`, `request-id` and `nonce` in the session.
 
+## Pre-Final Specifications
+
+Implementers should be aware that this specification uses several specifications that are not yet final specifications. Those specifications are:
+
+- OpenID Federation 1.0 draft -42 [@!OpenID.Federation]
+- SIOPv2 draft -13 [@!SIOPv2]
```

Comment:
If we mean draft 13, shouldn't the normative reference point to draft 13? i.e. https://openid.net/specs/openid-connect-self-issued-v2-1_0-13.html (the contents of which will never change)

Whereas it currently points to: https://openid.net/specs/openid-connect-self-issued-v2-1_0.html - the contents of which can change.

(I guess this comment may apply to other references too, I didn't check)

Reply:
good point

Reply:
I wondered about that as well, but openid connect core spec worked the same way: The reference was to latest (stable url to latest), but the note then specified specific draft versions with this note.

--> This is why i only adjusted the text here and left the references to latest.

Reply:
i think that's confusing and we should do better than OIDC?

Reply:
I would assume the intention was to reference the latest draft for the likely case that there are no breaking changes -> then it would be better to have a reference to the final specification.

I am happy to change this to whatever we decide if there is consensus which way to go

Reply:
> I wondered about that as well, but openid connect core spec worked the same way: The reference was to latest (stable url to latest), but the note then specified specific draft versions with this note.

The OIDC link points to 1.0 final. The only change that might happen to that are errata which should be non-breaking. It's up to us what we point to though, if we want to point to an explicit revision of OIDC (e.g. 1.0 errata 3) then we could.

Reply:
wg discussion:
- use URL to the version that is being referenced

Reply:
> wg discussion:
> 
> * use URL to the version that is being referenced

I think we reached a different consensus at the end of the discussion?

Reply:
i don't think so

Reply:
changed to fixed versions for drafts